src/routes.ts

```diff
@@ -21,7 +20,6 @@ export default [
   {
     path: '/',
```

Comment:
What happens now if I navigate to this path? Maybe we can remove this route?

Also need to think about default/fallback route at the bottom of this file...

Reply:
I think we will have a loop here if the user goes to route `/`, or an invalid route which also redirects to `/`.

One possibility is to create an error page if the route is invalid (eg, default/fallback route).

Another possibility is to redirect the user to the business' dashboard.

Reply:
Resolving as I will start a new conversation.